### PR TITLE
Replace queryByTestId with getByTestId

### PR DESCRIPTION
**Pull request by WhiteNight110**
I have completed the replacement of the corresponding code as required.

### CODE REVIEW

**Comment by eliserichards** [comment]
@WhiteNight110 thanks for taking on this task! It looks like linting is failing with the following errors: 

```
/experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx
  51:12  error  Replace `⏎······screen.getByTestId("label-enrollment-end-date"),⏎····` with `screen.getByTestId("label-enrollment-end-date")`  prettier/prettier
  89:12  error  Replace `⏎······screen.getByTestId("label-enrollment-end-date"),⏎····` with `screen.getByTestId("label-enrollment-end-date")`  prettier/prettier

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

You can fix those, and then run `make check` from the `experimenter` folder to see if there are any other errors to fix 👍

File: experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx [hunk 1]
```diff
@@ -26,15 +26,15 @@ describe("SummaryTimeline", () => {
     ].forEach((set) => {
       render(<Subject {...set} />);
 
-      expect(screen.queryByTestId("label-not-launched")).toBeInTheDocument();
-      expect(screen.queryByTestId("label-start-date")).not.toBeInTheDocument();
-      expect(screen.queryByTestId("label-release-date")).toBeNull();
+      expect(screen.getByTestId("label-not-launched")).toBeInTheDocument();
```
Comment (brennie) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx:29]: You don't need expect().toBeInTheDocument(). The call to `getByTestId()` will throw if it cant find the element.

File: experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx [hunk 2]
```diff
@@ -26,15 +26,15 @@ describe("SummaryTimeline", () => {
...
+      expect(screen.getByTestId("label-start-date")).not.toBeInTheDocument();
+      expect(screen.getByTestId("label-release-date")).toBeNull();
```
Comment (brennie) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx:31]: This is incorrect. getByTestId throws if the element is not found and we are asserting this element does not exist. Revert this change

File: experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx [hunk 3]
```diff
@@ -26,15 +26,15 @@ describe("SummaryTimeline", () => {
...
       expect(
-        screen.queryByTestId("label-enrollment-end-date"),
+        screen.getByTestId("label-enrollment-end-date"),
       ).not.toBeInTheDocument();
```
Comment (brennie) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx:34]: This too.

File: experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx [hunk 4]
```diff
@@ -26,15 +26,15 @@ describe("SummaryTimeline", () => {
...
-      expect(screen.queryByTestId("label-end-date")).not.toBeInTheDocument();
-      expect(screen.queryByTestId("label-duration-days")).toBeInTheDocument();
-      expect(screen.queryByTestId("label-enrollment-days")).toBeInTheDocument();
+      expect(screen.getByTestId("label-end-date")).not.toBeInTheDocument();
```
Comment (brennie) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx:35]: This too.

File: experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx [hunk 5]
```diff
@@ -46,14 +46,12 @@ describe("SummaryTimeline", () => {
...
 
-    expect(screen.queryByTestId("label-not-launched")).not.toBeInTheDocument();
-    expect(screen.queryByTestId("label-start-date")).toBeInTheDocument();
-    expect(
-      screen.queryByTestId("label-enrollment-end-date"),
-    ).toBeInTheDocument();
-    expect(screen.queryByTestId("label-end-date")).toBeInTheDocument();
-    expect(screen.queryByTestId("label-duration-days")).toBeInTheDocument();
-    expect(screen.queryByTestId("label-enrollment-days")).toBeInTheDocument();
+    expect(screen.getByTestId("label-not-launched")).not.toBeInTheDocument();
```
Comment (brennie) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx:49]: This too.

File: experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx [hunk 6]
```diff
@@ -84,23 +82,21 @@ describe("SummaryTimeline", () => {
...
```
Comment (brennie) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx:85]: This too.

**Review by brennie — CHANGES_REQUESTED** [review]

**Comment by WhiteNight110** [comment]
Sorry, I have been busy lately and may not be able to continue working

**Comment by jaredlockhart** [comment]
@WhiteNight110 No problem, thank you for taking a stab at this, I will close this and if you get time in the future feel free to come back and reopen it or file a new PR 🙏

**Pull request closed without merging**